packages/lesswrong/server/repos/UsersRepo.ts

```diff
@@ -172,7 +173,6 @@ class UsersRepo extends AbstractRepo<"Users"> {
 
   private getSearchDocumentQuery(): string {
     return `
-      -- UsersRepo.getSearchDocumentQuery
```

Comment:
Could you add this back in?

Reply:
I'm pretty sure it's just a mistake - this function doesn't run a query, it just returns a string that gets inserted into other queries, and those other queries add their own comments.

Reply:
Ah right my bad